### DISCUSSION TITLE
怎么注销账号·永久删除账号？How to cancel an account and permanently delete an account?

### DISCUSSION THREAD

**Discussion by z8585** [community/community#140869] [Other Feature Feedback, Questions, & Ideas]
怎么注销账号·永久删除账号？
How to cancel an account and permanently delete an account?

**Comment by davevad93** [comment]
Hi @z8585 ,

Go to your [account settings](https://github.com/settings/admin), scroll down to the "***Delete account***" section and click `Delete your account`.

**Reply by z8585** [reply]
I saw this logout function in the settings. If I logout, how long will it take to release this account?

**Reply by Hugolobo22** [reply]
Go to Account settings -> Account (Gear icon) -> scroll down until you see the button "Delete Account" -> Click on it

**Comment by Akash1134** [comment]
Welcome to the GitHub Community, @z8585 , we're happy you're here! I've moved your post to the right category.

**Comment by Hugolobo22** [comment]
Go to Account settings -> Account (Gear icon) -> scroll down until you see the button "Delete Account" -> Click on it

**Reply by z8585** [reply]
After clicking, how long will it take for it to be released，It refers to the binding of accounts, email addresses, etc. How long will it be released?How long does it take to delete an account?

**Reply by Hugolobo22** [reply]
After clicking the Delete button, complete the steps in the dialog box to confirm you understand what will happen instantly, i think
You can view better here: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account

**Reply by z8585** [reply]
ok
